applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -842,6 +843,72 @@ static struct bt_conn_cb conn_callbacks = {
 
 };
 
+/**
+ * @brief	Callback handler for play/pause.
+ *
+ * @note	This callback is called from MCS after receiving a
+ *		command from the client or the local media player.
+ *
+ * @param[in]	play  Boolean to indicate if the stream should be resumed or paused.
+ */
+static void le_audio_play_pause_cb(bool play)
+{
+	int ret;
+
+	LOG_DBG("Play/pause cb, state: %d", play);
+
+	if (play) {
+		playing_state = true;
```

Comment:
Should "playing_state = true;" be moved to after the bt_audio_stream_enable() function call(s)?

Reply:
As we discussed, I will create a helper function too get the playing stated from active headsets

Reply:
Create a separate task for this?

Reply:
Done